packages/fiber/package.json

```diff
@@ -44,12 +44,12 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.17.8",
-    "@types/react-reconciler": "^0.28.9",
+    "@types/react-reconciler": "^0.32.0",
     "@types/webxr": "*",
     "base64-js": "^1.5.1",
     "buffer": "^6.0.3",
     "its-fine": "^2.0.0",
-    "react-reconciler": "^0.31.0",
+    "react-reconciler": "^0.32.0",
```

Comment:
This is a breaking change, as `react-reconciler` corresponds to React 19.1.

Been thinking on this. We could choose to inline if we're sure it is backwards compatible.

Reply:
That's a good catch. I'll verify this.

Reply:
I confirmed `flushSyncFromReconciler` is in v0.31 as well. I think it is safe to use it without inlining. We now have a test for `flushSync` so going forward we can know if this breaks when updating the reconciler. I'll downgrade the minimum version though as you pointed out.